shacl12-core/index.html

```diff
@@ -2700,7 +2701,7 @@ <h3>Conformance Checking</h3>
 					<a>conforms</a> to a given <a>shape</a>, and <code>false</code> otherwise.
 				</p>
 				<p id="conformance-nested">
-					Note that some <a>constraint components</a> of SHACL Core (e.g., those of <code>sh:not</code>, <code>sh:or</code> and <code>sh:node</code>)
+					Note that some <a>constraint components</a> of SHACL Core (e.g., those of <code>sh:not</code>, <code>sh:or</code>, <code>sh:node</code>, and <code>sh:memberShape</code>)
```

Comment:
_(Non-blocking suggestion improvement, with follow-on message I suspect will migrate to an Issue.)_

Now that this line's highlighted, I just noticed a missing symmetry.  If this parenthetical is meant to be exhaustive, shouldn't `sh:and` and `sh:xone` be here too?

Going a bit further, so we can pare back:  The list of "shape-expecting constraint parameters" is on line 2645, and 8 items long.

I think we can shorten some English here and remove the parenthetical.  Couldn't we just say:

> Note that all shape-expecting constraint parameters [link to L2645] of SHACL Core rely on conformance checking.

I suggest that because, unless I've forgotten something, all 8 of those constraint components rely on conformance checking.

Reply:
Change made in https://github.com/w3c/data-shapes/pull/403/commits/180b418d55c3995347a4625cd4a7a588b8b85c78. I think the wording "shape-expecting constraint parameters" is correct noting the [following from the spec](https://www.w3.org/TR/shacl/#constraints)

> A constraint component is an [IRI](https://www.w3.org/TR/shacl/#dfn-iri). Each constraint component has one or more mandatory parameters, each of which is a property. Each constraint component has zero or more optional parameters, each of which is a property. The parameters of a constraint component are its mandatory parameters plus its optional parameters.